src/MusicStore/Areas/Admin/Controllers/StoreManagerController.cs

```diff
@@ -130,19 +146,19 @@ public IActionResult Create()
         // POST: /StoreManager/Edit/5
         [HttpPost]
         [ValidateAntiForgeryToken]
-        public async Task<IActionResult> Edit(Album album)
+        public async Task<IActionResult> Edit(CancellationToken requestAborted, Album album)
```

Comment:
Yes..I haven't updated it yet..I was waiting for more comments, but I'll update as you suggested now..